pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -188,6 +190,13 @@ def load_data(self):
             )
         )
 
+    def load_uploads(self):
+        """
+        Copies upload fixtures into MEDIA_ROOT
+        """
+        print 'Copying fixtures from uploads directory to MEDIA_ROOT'
+        sh('cp -rf common/test/uploads/* test_root/uploads/')
```

Comment:
The reason for copying these files over here is that `test_root/uploads/` is the `MEDIA_ROOT` for the bokchoy tests, and pretty much everything that ends up there is included in the `.gitignore`. So I figured it would make sense to put the fixtures somewhere else (`common/test/` is where the other fixtures live) and copy them over when the tests are run.

I can copy them over using python rather than the shell if you prefer.

I would prefer to reference the `MEDIA_ROOT` setting here, but we don't have access to it yet as we haven't loaded the django settings.

We could avoid this altogether by loading these fixtures somewhere in the test setup, rather than by using `./manage.py loaddata`.